### PR TITLE
Create alon-lavie.md

### PR DESCRIPTION
**Pull request by ovesyan**

### CODE REVIEW

File: people/alon-lavie.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Alon Lavie
+Alon Lavie is a machine translation researcher. He received Ph.D. in computer science at Carnegie Mellon University in 1995. 
+
+His main research interests are Machine Translation adaptation approaches, automatic metrics for machine translation 
+evaluation, quality estimation, and methods for multi-engine machine translation system combination.
+
+Lavie served as president at the International Association for Machine Translation and the Association for Machine 
+Translation in the Americas. He was also a member of the Association for Computational Linguistics (ACL).  
+
+In 1996-2015, Lavie worked as a research professor at the Language Technologies Institute at Carnegie Mellon University.
```
Comment (bittlingmayer) [inline people/alon-lavie.md:13]: No dashes

File: people/alon-lavie.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+In 2004, Lavie together with Michael J. Denkowski developed and released [METEOR](/metric/meteor.md), an automatic metric 
+for the evaluation of machine translation output. 
+
+In 2009, he co-founded Safaba Translation Solutions that offered automated translation solutions to global enterprises. 
+In 2015, it was aquired by Amazon. 
+
+In 2015-2019, Lavie was a senior manager at Amazon, where he led and managed the Amazon Machine Translation R&D group in Pittsburgh. 
+
+In 2019, he moved to Unbabel as a vice president of Language Technologies in charge of leading and managing US AL lab based in Pittsburgh,
```
Comment (bittlingmayer) [inline people/alon-lavie.md:23]: Vice President is a title, so titlecase

Comment (bittlingmayer) [reply people/alon-lavie.md:23]: > in charge of leading and managing

3 ways of saying the same thing ;-)

> to lead the AI lab.

but I don't think it's totally necessary.

File: people/alon-lavie.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+In 2004, Lavie together with Michael J. Denkowski developed and released [METEOR](/metric/meteor.md), an automatic metric 
+for the evaluation of machine translation output. 
+
+In 2009, he co-founded Safaba Translation Solutions that offered automated translation solutions to global enterprises. 
+In 2015, it was aquired by Amazon. 
+
+In 2015-2019, Lavie was a senior manager at Amazon, where he led and managed the Amazon Machine Translation R&D group in Pittsburgh. 
+
+In 2019, he moved to Unbabel as a vice president of Language Technologies in charge of leading and managing US AL lab based in Pittsburgh, 
+and of providing strategic leadership for AI R&D teams world-wide.
```
Comment (bittlingmayer) [inline people/alon-lavie.md:24]: Es el karanq toxenq, petq chi

File: people/alon-lavie.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+description: Machine translation researcher and co-developer of METEOR
+---
+# Alon Lavie
+Alon Lavie is a machine translation researcher. He received Ph.D. in computer science at Carnegie Mellon University in 1995. 
+
+His main research interests are Machine Translation adaptation approaches, automatic metrics for machine translation
```
Comment (bittlingmayer) [inline people/alon-lavie.md:7]: Bullet points

Comment (bittlingmayer) [reply people/alon-lavie.md:7]: *machine translation* should not be title case

**Comment by ovesyan** [comment]
I am not sure I've added the links correctly.

File: people/alon-lavie.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Lavie served as president at the International Association for Machine Translation and the Association for Machine 
+Translation in the Americas. He was also a member of the Association for Computational Linguistics (ACL).  
+
+From 1996 to 2015, he worked as a research professor at the Language Technologies Institute at Carnegie Mellon University.
+
+In 2004, Lavie together with Michael J. Denkowski developed and released [METEOR](/metrics/meteor.md), an automatic metric 
+for the evaluation of machine translation output. 
+
+In 2009, he co-founded [Safaba](/companies/safaba.md). In 2015, it was acquired by Amazon, and became Amazon Translate.
```
Comment (bittlingmayer) [inline people/alon-lavie.md:21]: `/business/companies#safaba`

**Pull request merged**